routers/repo/view.go

```diff
@@ -119,6 +120,21 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 	ctx.Data["LatestCommit"] = latestCommit
 	ctx.Data["LatestCommitVerification"] = models.ParseCommitWithSignature(latestCommit)
 	ctx.Data["LatestCommitUser"] = models.ValidateCommitWithEmail(latestCommit)
+	commit := models.SignCommitWithStatuses{
```

Comment:
Good point. I took some portion of code from the display. I will use local variable instead.